spec/values/color.hrx

```diff
@@ -0,0 +1,47 @@
+<===> keyword/input.scss
```

Comment:
I think these specs should be substantially expanded:

* We should verify that these values are actually parsed as colors, not just that they're emitted as written. Maybe pass them to `adjust-color()` with no other args, which will both verify that its argument is a color and return a color value without an associated string representation?

* We need to all color names Sass supports, and that those names are case-insensitive.

* We need to verify that hex digits are parsed correctly into red/green/blue/alpha channels for all hex representations.